.github/CONTRIBUTING.md

```diff
@@ -88,9 +96,9 @@ To run linter/style checker:
 
 ## Reporting a Security Issue
 
-If you've found a security related issue in CCR, please don't open an issue in github. Instead contact us at adamsb@msu.edu. 
+If you've found a security related issue in CCR, please don't open an issue in github. Instead contact us at mesh@msu.edu.
```

Comment:
I would recommend adding a name (person, organization) for reporters to address the email to (ex: "Dear MESH").

Reply:
Addressed in an additional commit.